src/dmell_ia.c

```diff
@@ -11,11 +59,261 @@ static void print_prompt()
 {
     const char* host_name = Dmod_GetEnv( "HOSTNAME" );
     host_name = ( host_name != NULL ) ? host_name : "dmell";
-    char cwd[256] = {0};
+    char cwd[1024] = {0};
```

Comment:
z tym to sobie jaja robisz?! 1kB na takie gówno na embedded? I to jeszcze na stosie?